gun.js

```diff
@@ -1250,6 +1250,9 @@
 				}
 				k && (to.path || (to.path = [])).push(k);
 				if(!(v = valid(d)) && !(g = Gun.is(d))){
+          if(as.array && d instanceof Array) {
```

Comment:
this file is built from files in src, so you're going to have to add this change in `src/put.js` too?

Reply:
good call, thanks, added!